phoenix-core/src/main/java/org/apache/phoenix/iterate/RegionScannerFactory.java

```diff
@@ -247,6 +242,11 @@ public boolean nextRaw(List<Cell> result) throws IOException {
             }
           }
           if (projector != null) {
+            Cell arrayElementCell = null;
```

Comment:
This actually the only place where this was needed, and it is done here after we re-positioned the local index scan (scanTillStarRow) and after uncovered columns are merged in.